les/server.go

```diff
@@ -50,7 +50,7 @@ type ethBackend interface {
 	BloomIndexer() *core.ChainIndexer
 	ChainDb() ethdb.Database
 	Synced() bool
-	TxPool() *txpool.TxPool
+	TxPool() miner.TxPool
```

Comment:
What's behind the change in TxPool here?

Reply:
honestly, it was kind of unnecessary in the end, my idea was to make it an interface so we can have a normal geth tx pool implementation, and an astria-specific one. but we'd end up having to implement a lot more functions than I thought. can revert since it doesn't actually do anything

Reply:
reverted the change!